src/index.css

```diff
@@ -75,3 +97,36 @@ body {
     cursor: pointer;
     color: #ccc;
 }
+
+.tv-show-seasons {
```

Comment:
dla czystości i porządku nie trzymamy pustych i nieużywanych reguł w pliku. Wyjeb. Jak zaczniesz potrzebować to się dopisze